### ISSUE TITLE
The display of the form does not actually match the display in the documentation.

### ISSUE THREAD

**Issue by AlexElizard** [LeeHanYeong/django-quill-editor#31]
The display of the form does not actually match the display in the documentation. Is that how it should be?
I have:
* Python 3.9.1
* Django 3.1.4

![image](https://user-images.githubusercontent.com/52967658/103180761-aac96200-48e4-11eb-81a8-c7e943a4903b.png)

**Comment by sxwild** [comment]
Yes, I have the same issue. I have the same issue, some Quill Fields show as per the documentation and some as shown above. This happens even in the same form. The way it looks as documented is much neater.

**Comment by aritztg** [comment]
I have the same issue. I was expected the editor in the right column, not aligned to the left as @sxwild shows in the capture. Does anyone the reason for this?
